tensorboard/plugins/hparams/backend_context.py

```diff
@@ -75,10 +75,38 @@ def experiment(self, experiment_id):
           protobuffer can be built (possibly, because the event data has not been
           completely loaded yet), returns None.
         """
-        experiment = self._find_experiment_tag(experiment_id)
-        if experiment is None:
-            return self._compute_experiment_from_runs(experiment_id)
-        return experiment
+        return self.experiment_from_metadata(
```

Comment:
OK: There’s only one more caller of `experiment`, so it’s fine with me
to either leave this as is or inline it into `get_experiment.py`.